From d60831f10df7969ca4bc5d3aade7be99a9caa949 Mon Sep 17 00:00:00 2001
From: crazy <crazy@frugalware.org>
Date: Fri, 7 Mar 2008 23:03:57 +0100
Subject: [PATCH] configure.ac
 * prepare for bump party ;)
 * changed CHOST to ${CARCH}-frugalware-linux

---
 configure.ac |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index ac2d49b..936dde1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -96,7 +96,7 @@ case "$host" in
 		CARCH="i686"
 		CARCHFLAGS="i686"
 		MARCHFLAG="march"
-		CHOST="${host}"
+		CHOST="${CARCH}-frugalware-linux"
 		;;
 	x86_64-*) AC_DEFINE_UNQUOTED([CARCH], ["x86_64"], [Architecture])
 		AC_DEFINE_UNQUOTED([CARCHFLAGS], ["k8"], [Architecture])
@@ -104,7 +104,7 @@ case "$host" in
 		CARCH="x86_64"
 		CARCHFLAGS="k8"
 		MARCHFLAG="march"
-		CHOST="${host}"
+		CHOST="${CARCH}-frugalware-linux"
 		;;
 	ia64-*) AC_DEFINE_UNQUOTED([CARCH], ["ia64"], [Architecture])
 		AC_DEFINE_UNQUOTED([CARCHFLAGS], ["ia64"], [Architecture])
@@ -112,7 +112,7 @@ case "$host" in
 		CARCH="ia64"
 		CARCHFLAGS="ia64"
 		MARCHFLAG="march"
-		CHOST="${host}"
+		CHOST="${CARCH}-frugalware-linux"
 		;;
 	sparc-*) AC_DEFINE_UNQUOTED([CARCH], ["sparc"], [Architecture])
 		AC_DEFINE_UNQUOTED([CARCHFLAGS], ["v9"], [Architecture])
@@ -120,7 +120,7 @@ case "$host" in
 		CARCH="sparc"
 		CARCHFLAGS="v9"
 		MARCHFLAG="mcpu"
-		CHOST="${host}"
+		CHOST="${CARCH}-frugalware-linux"
 		;;
 	ppc-* | powerpc-*)
 		AC_DEFINE_UNQUOTED([CARCH], ["ppc"], [Architecture])
@@ -129,7 +129,7 @@ case "$host" in
 		CARCH="ppc"
 		CARCHFLAGS="750"
 		MARCHFLAG="mcpu"
-		CHOST="${host}"
+		CHOST="${CARCH}-frugalware-linux"
 		;;
 	i386-*) AC_DEFINE_UNQUOTED([CARCH], ["i386"], [Architecture])
 		AC_DEFINE_UNQUOTED([CARCHFLAGS], ["i386"], [Architecture])
@@ -137,7 +137,7 @@ case "$host" in
 		CARCH="i386"
 		CARCHFLAGS="i386"
 		MARCHFLAG="march"
-		CHOST="${host}"
+		CHOST="${CARCH}-frugalware-linux"
 		;;
 	*)
 		AC_MSG_ERROR([Your architecture is not supported])
-- 
1.5.4.3

